continuous_integration/travis_install.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+set -ex
+
+PY_VERSION=$TRAVIS_PYTHON_VERSION
+WHEELHOUSE_URI=https://parkin.github.io/python-wheelhouse/
+
+install_conda()
+{
+  # Define the value to download
+  if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+    MINICONDA_OS=$MINICONDA_LINUX;
+  elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
+    MINICONDA_OS=$MINICONDA_OSX;
+  fi
+
+  # You may want to periodically update this, although the conda update
+  # conda line below will keep everything up-to-date.  We do this
+  # conditionally because it saves us some downloading if the version is
+  # the same.
+
+  if [ "$PY_VERSION" = "2.7" ]; then
+    wget "http://repo.continuum.io/miniconda/Miniconda-$MINICONDA_VERSION-$MINICONDA_OS.sh" -O miniconda.sh;
+  else
+    wget "http://repo.continuum.io/miniconda/Miniconda3-$MINICONDA_VERSION-$MINICONDA_OS.sh" -O miniconda.sh;
+  fi
+
+  bash miniconda.sh -b -p "$HOME/miniconda";
+  export PATH="$HOME/miniconda/bin:$PATH";
+  hash -r;
+  conda config --set always_yes yes --set changeps1 no;
+  conda update -q conda;
+
+  # Useful for debugging any issues with conda
+  conda info -a;
+
+  # Test environments for different Qt bindings
+  if [ "$USE_QT_API" = "PyQt5" ]; then
+    #sudo apt-get install -q "^libxcb.*" libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev
+    #sudo apt-get install -q libxcb-sync0-dev libxcb-render-util0 libxcb-image0  libxcb-xfixes0 libxcb-randr0 libxcb-keysyms1
+    #sudo cp- /usr/lib/libxcb-render-util.so.0 /usr/lib/x86_64-linux-gnu/libxcb-render-util.so.0
+    
+    sudo apt-get install -q "^libxcb.*" libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev
+    #sudo ln -sf /usr/lib/x86_64-linux-gnu/libxcb-render-util.so.0 /usr/lib/libxcb-render-util.so.0 
+
+    conda config --add channels dsdale24;
+    conda create -q -n test-environment python=$PY_VERSION sphinx sip qt5 pyqt5;
+    ldd "$HOME/miniconda/envs/test-environment/lib/qt5/plugins/platforms/libqxcb.so"
+
+    # libxcb-atom1-dev libxcb-event1-dev libxcb-icccm1-dev
+    # ldd /home/goanpeca/anaconda/envs/test-environment/lib/qt5/plugins/platforms/libqxcb.so
+    #	
+
+  elif [ "$USE_QT_API" = "PyQt4" ]; then
+    conda create -q -n test-environment python=$PY_VERSION sphinx sip qt pyqt;
```

Comment:
We should include here more packages. At least `ipython-qtconsole`, `matplotlib` and `pandas`.